lib/flipper/engine.rb

```diff
@@ -23,6 +23,10 @@ class Engine < Rails::Engine
     end
 
     initializer "flipper.default", before: :load_config_initializers do |app|
+      # Load cloud secrets from Rails credentials
+      ENV["FLIPPER_CLOUD_TOKEN"] ||= app.credentials.dig(:flipper, :cloud_token)
```

Comment:
I assume we're comfortable assuming all instances of `app` will have `credentials`? I couldn't find any official details on which versions of Rails are supported. Maybe worth making this more resilient just to be safe?

Reply:
I updated the test to actually write the credentials instead of stubbing them just to make sure.